app/uk/gov/hmrc/awrslookup/controllers/LookupController.scala

```diff
@@ -72,22 +69,17 @@ class LookupController @Inject()(mcc: MessagesControllerComponents,
     )
   }
 
-
-  def show(fromMulti: Boolean = false): Action[AnyContent] = Action.async {
+  def show(): Action[AnyContent] = Action.async {
 
     implicit request =>
 
       val lang: Lang = request.lang
-      val action = controllers.routes.LookupController.show(fromMulti)
-      (request.queryString.get(SearchForm.query).isDefined, request.queryString.get("originalSearchTerm").isDefined) match {
-        case (true, true) =>
-          validateFormAndSearch(preValidationForm = searchForm, action = action, lookupCall = lookupService.lookup, fromMulti = fromMulti, originalSearchTerm = Some(request.queryString.get("originalSearchTerm").get.head))
-        case (true, false) =>
-          validateFormAndSearch(preValidationForm = searchForm, action = action, lookupCall = lookupService.lookup, fromMulti = fromMulti, originalSearchTerm = None)
-        case _ => Ok(searchMain(searchForm.form, action)(request, request2Messages, messagesApi, lang))
+      val action = controllers.routes.LookupController.show()
+      if (request.queryString.contains(SearchForm.query)) {
```

Comment:
Unsure about this one - not sure what the functionality of originalSearchTerm is, will need to look into it further